modules/oadp-installing-dpa.adoc

```diff
@@ -150,6 +171,11 @@ spec:
           project: <project>
           snapshotLocation: <us-west1> <.>
 ----
+<.> Set to `false` if you want to disable the Restic installation. Restic deploys a daemon set, which means that an environment with many nodes will have a lot of `Restic` pods running. You configure Restic for backups by adding `spec.defaultVolumesToRestic: true` to the `Backup` CR.
```

Comment:
`Restic deploys a daemon set`  --> `Restic deploys a DaemonSet`

Reply:
`which means that an environment with many nodes will have a lot of `Restic` pods running.`
-->
`which means that an environment with each worker nodes will have a `Restic` pods running.`